helm/chaos-mesh/templates/chaos-daemon-rbac.yaml

```diff
@@ -58,16 +58,45 @@ metadata:
     {{- include "chaos-mesh.labels" . | nindent 4 }}
     app.kubernetes.io/component: chaos-daemon
 spec:
-  privileged: false
-  allowPrivilegeEscalation: true
+  {{- if .Values.chaosDaemon.privileged }}
+  allowedCapabilities:
+    - SYS_PTRACE
+  {{- else }}
   allowedCapabilities:
-    - '*'
+    - SYS_PTRACE
+    - NET_ADMIN
+    - MKNOD
+    - SYS_CHROOT
+    - SYS_ADMIN
+    - KILL
+    # CAP_IPC_LOCK is used to lock memory
+    - IPC_LOCK
+  {{- end }}
+  allowedHostPaths:
+    - pathPrefix: {{ .Values.chaosDaemon.socketPath | default "/var/run/docker.sock" }}
```

Comment:
that default value is not necessary, `.Values.chaosDaemon.socketPath` always exists.